src/sonic-yang-models/tests/files/sample_config_db.json

```diff
@@ -682,6 +682,39 @@
                 "nexthop_group_threshold_type": "percentage",
                 "polling_interval": "0"
             }
+        },
```

Comment:
Nat zone is not needed in yang model?

Reply:
Yes, added NAT zone for INTERFACE, VLAN_INTERFACE and LOOPBACK_INTERFACE tables.